### PR TITLE
feat: BREAKING route/header ILP addresses

### PR DESCRIPTION
**Pull request by sentientwaffle**

### CODE REVIEW

File: schemas/IlpHeader.json [hunk 1]
```diff
@@ -6,11 +6,11 @@
   "properties": {
     "ledger": {
       "description": "The ledger where the transfer will take place",
-      "$ref": "Iri.json"
+      "type": "string"
```
Comment (emschwartz) [inline schemas/IlpHeader.json]: Is `ledger` still needed in addition to `account`?

File: schemas/IlpHeader.json [hunk 2]
```diff
@@ -6,11 +6,11 @@
...
     },
     "account": {
       "description": "Account holding the funds",
-      "$ref": "Iri.json"
+      "type": "string"
```
Comment (emschwartz) [inline schemas/IlpHeader.json]: I think we should have an `Address` schema that gives the type as well as a regex

Comment (sentientwaffle) [inline schemas/IlpHeader.json]: I agree; what constraints are there on an ILP address?

File: schemas/IlpHeader.json (hunk 1 above)
Comment (sentientwaffle) [inline schemas/IlpHeader.json]: https://github.com/interledger/five-bells-connector/blob/febb7bf2680f22381ff58778f938be86d3820970/src/lib/route-builder.js#L103-L106

Comment (sentientwaffle) [inline schemas/IlpHeader.json]: Actually, that can probably be changed to use `account` and just strip off the last section.

File: schemas/IlpHeader.json (hunk 2 above)
Comment (justmoon) [inline schemas/IlpHeader.json]: Undecided - the main two options are:

* Any OCTET STRING, i.e. no restriction at all - in that case the JSON version would likely be base64url encoded, so you'd have [a-zA-Z0-9_-] as the regex.
* IA5 (aka ASCII) or some subset.

For right now, let's just go with [url-safe characters](https://tools.ietf.org/html/rfc3986#section-2.3) - `/[a-zA-Z0-9._~-]*/`.

**Comment by justmoon** [comment]
LGTM

**Pull request merged**